### ISSUE TITLE
Improve design for the Post page

### ISSUE THREAD

**Issue by dfeyer** [Plume-org/Plume#420]
As posted on riot this afternoon I start working on some design research around the UI.

# Goals

- Add a bit more personality
- Solve low contrast issues (accessibility)
- Improve the relation between form element and button 
- Respect the current slim and calm design
- Respect the current color palette

# Link to the figma project

The Figma design use the Noto font, because the font used by Plume are not support in Figma. My research are not around the font, and the current fonts are good.

You can follow my journey here:
https://www.figma.com/file/CZ6IxnQgYGhy5H4WYr1lA6VK/UI?node-id=0%3A1

# Changes

- [x] More graphical page header (the idea is to have the image on full screen, over the fold)
- [x] More "like & share" widget before the author area (it's related to the content, not the author)
- [x] Redesign a bit the page footer
- [x] Try to be a bit more casual in the "like & share" widget
- [x] Redesign the commenting form (improve difference between input & button)
- [x] Small redesign of the tag list
- [ ] Works on the responsive version

You can comments here, or directly on Figma

Hope that this little refresh make readers happy.

**Comment by Granitosaurus** [comment]
I'm not sure if this the right issue to address this but I'd also like to address the design of comments part of the page. 

It's not very structured and has too much white space as result it's pretty hard to follow and read.
e.g. just disabling some margins already make a world of difference imo:
![no margin](https://i.imgur.com/viQlHtx.png)

**Comment by ahangarha** [comment]
Header and footer and also so-called modern editor are not following responsive design. Should I file a new issue for it or it can be here?
